monitoring/interpod.go

```diff
@@ -359,7 +362,7 @@ func waitForAllNodesSchedulable(ctx context.Context, c *kube.Clientset) (nodes *
 			ResourceVersion: "0",
 			FieldSelector:   fields.Set{"spec.unschedulable": "false"}.String(),
 		}
-		nodes, err = c.CoreV1().Nodes().List(opts)
+		nodes, err = c.CoreV1().Nodes().List(context.TODO(), opts)
```

Comment:
`waitForAllNodesSchedulable` has a context arg.  Should this inherit from general context for the function?

Reply:
I will rewrite all that context.TODO into context from param of function. That was POC change to check if the installation of K8s is working.